pages/instructorEffectiveUser/instructorEffectiveUser.js

```diff
@@ -19,6 +19,11 @@ router.get('/', function(req, res, next) {
     sqldb.queryOneRow(sql.select, params, function(err, result) {
         if (ERR(err, next)) return;
         _.assign(res.locals, result.rows[0]);
+        res.locals.ipaddress = req.ip;
```

Comment:
Our current reverse-proxy setup sends proxy IP as `req.ip` and puts the real IP in the `x-forwarded-for` header. Our current method for getting the real IP is:
https://github.com/PrairieLearn/PrairieLearn/blob/master/middlewares/authzCourseInstance.js#L18

Reply:
I had thought we had `app.set('trust proxy', true);` added to PL, but maybe not? That's the magic to make Express do the proxying translations for us. http://expressjs.com/en/guide/behind-proxies.html